### DISCUSSION TITLE
Query Vessel names from Postgre database

### DISCUSSION THREAD

**Discussion by rkarikari** [jvde-github/AIS-catcher#168] [General]
Hello 
Is it possible for ais-catcher  to make use of the Postgre SQL database which holds a list of ship names to MMSI info ?
This will enable the immediate  resolution of shipnames  ( if it exists in the database ) as is the case with 
the ais aggregators like ...

MarineTraffic
Vesselfinder
Shipxplorer etc.

**Comment by jvde-github** [comment]
Hi, at the moment this is not possible. Perhaps some day but will require some work..... 

Thanks for the idea!

Jasper
